store/tikv/gcworker/gc_worker.go

```diff
@@ -515,19 +515,51 @@ func doGC(ctx goctx.Context, store tikv.Storage, safePoint uint64, identifier st
 	// Sleep to wait for all other tidb instances update their safepoint cache.
 	time.Sleep(gcSafePointCacheInterval)
 
+	log.Infof("[gc worker] %s start gc, safePoint: %v.", identifier, safePoint)
+	startTime := time.Now()
+	regions := 0
+	errRegions := 0
+
+	bo := tikv.NewBackoffer(tikv.GcOneRegionMaxBackoff, goctx.Background())
+	var key []byte
+	for {
+		select {
+		case <-ctx.Done():
+			return errors.New("[gc worker] gc job canceled")
+		default:
+		}
+
+		loc, err := store.GetRegionCache().LocateKey(bo, key)
+		if err != nil {
+			return errors.Trace(err)
+		}
+
+		if err = doGCForOneRegion(ctx, store, safePoint, identifier, loc.StartKey, loc.EndKey); err != nil {
+			regions++
+		} else {
+			errRegions++
+		}
```

Comment:
It is `errRegions` need ++ when `err != nil`.

Reply:
ok